SPECS/kernel-mshv/50_mariner_mshv_menuentry

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
```

Comment:
note: this actually does need the shebang because grub2-mkconfig calls them as arbitrary bins (no './')

![image](https://github.com/microsoft/CBL-Mariner/assets/37827103/698192ed-7fb4-4e09-8868-82a744945be9)